components/new-contribution-flow/StepPayment.js

```diff
@@ -117,7 +118,9 @@ const StepPayment = ({
 
   return (
     <Container width={1} border={['1px solid #DCDEE0', 'none']} borderRadius={15}>
-      {loading && !paymentMethods.length ? (
+      {stepProfile.blocked ? (
+        <BlockedContributorMessage categories={stepProfile.blocked} collective={collective} />
```

Comment:
Weird to pass "blocked" as "categories. It's a hint that `blocked` is not properly named. `rejectedCategories`? `contributorRejectedCategories`?